moped-database/migrations/1668201706589_create_phase_key/up.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE moped_phases ADD COLUMN phase_key text UNIQUE;
```

Comment:
i'm open to ideas for a better name for this column - and if we need it at all. in the PR that follows (#872) i think makes for cleaner editor code.

Reply:
thinking.....

Reply:
yep, i think that's right. that's what we use on `moped_tags` for a column that has essentially the same purpose. my only hesitation is that slug conveys a string with dashes (`-`) and i'm partial to snake case these values are used as object accessors.

Reply:
just realizing i forgot to follow-up on this column name. it would be a small lift to change this and it would be best to do it sooner than later. let me know how this is sitting with y'all :)